video2dataset/worker.py

```diff
@@ -159,12 +155,19 @@ def data_generator():
                         semaphore.release()
                         continue
 
-                    bytes_downloaded += len(vid_stream)
+                    if vid_stream is not None:
+                        bytes_downloaded += len(vid_stream)
+                    if aud_stream is not None:
+                        bytes_downloaded += len(aud_stream)
```

Comment:
not sure this is correct if you extract the audio from the video that means the bytes of the audio are in the video. don't duplicate them